feat: UTC-205: Reduce the number of users fetched for the Annotators and Reviewers

mcanu opened the pull request:
<!--

This description MUST be filled out for a PR to receive a review. Its primary purposes are:

 - to enable your reviewer to review your code easily, and
 - to convince your reviewer that your code works as intended.

Some pointers to think about when filling out your PR description:
 - Reason for change: Description of problem and solution
 - Screenshots: All visible changes should include screenshots.
 - Rollout strategy: How will this code be rolled out? Feature flags / env var / other
 - Testing: Description of how this is being verified
 - Risks: Are there any known risks associated with this change, eg to security or performance?
 - Reviewer notes: Any info to help reviewers approve the PR
 - General notes: Any info to help onlookers understand the code, or callouts to significant portions.

You may use AI tools such as Copilot Actions to assist with writing your PR description (see https://docs.github.com/en/copilot/using-github-copilot/using-github-copilot-for-pull-requests/creating-a-pull-request-summary-with-github-copilot); however, an AI summary isn't enough by itself. You'll need to provide your reviewer with strong evidence that your code works as intended, which requires actually running the code and showing that it works.

-->

mcanu committed:
Remove annotators manager annotations

nass600 commented:
/git merge
> [Workflow run](https://github.com/HumanSignal/label-studio/actions/runs/16572395785)
> Successfully merged:  create mode 100644 web/libs/ui/src/assets/icons/cloud-provider-s3.svg

nass600 committed:
Merge branch 'develop' into 'fb-utc-205'

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/16572395785

mcanu committed:
Undo changes

nass600 committed:
removed main users request and hydrating from the endpoint

mcanu committed:
Removed static_method from prefetch

nass600 committed:
restored status badges

robot-ci-heartex committed:
ci: Build tag docs

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/16597955580

nass600 committed:
added annotator_count

nass600 committed:
all counters added

mcanu committed:
Using task serializer context with queryset

wesleylima commented on web/libs/datamanager/src/components/CellViews/Annotators/Annotators.jsx:
```diff
@@ -79,25 +98,61 @@ const UsersInjector = inject(({ store }) => {
 });
 
 Annotators.filterItems = (items) => {
+  console.log("filters", items);
```
Do we want this?

nass600 replied:
wip code... thanks amte

nass600 committed:
cleaning up filters

nass600 committed:
simplified user management

nass600 committed:
added max users limit to fe

nass600 committed:
Merge remote-tracking branch 'origin/develop' into fb-utc-205

nass600 committed:
removed todos

nass600 committed:
fixing label stream

nass600 committed:
fixing assignee

mcanu committed:
Fix test

mcanu commented:
/fm sync
> [Workflow run](https://github.com/HumanSignal/label-studio/actions/runs/16651206815)

mcanu committed:
Sync Follow Merge dependencies

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/16651206815

mcanu committed:
Merge branch 'develop' into 'fb-utc-205'

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/16651206815

mcanu commented:
/fm sync
> [Workflow run](https://github.com/HumanSignal/label-studio/actions/runs/16654066135)

mcanu committed:
Sync Follow Merge dependencies

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/16654066135

mcanu committed:
Merge branch 'develop' into 'fb-utc-205'

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/16654066135
